### PR TITLE
fix: Fixed rounded corner misalignment in media container

### PR DESCRIPTION
**Pull request by at-susie**
### Description
We discovered that there is a small gap in rounded corner sizes in Container with media variant.
rounded-corner-gap

Related links, issue #, if available: n/a

### How has this been tested?

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/container/styles.scss
Comment (just-boris) [inline src/container/styles.scss:14]: We cannot do this, because it will make absolute elements (dropdowns and popover) cropped by the container borders.

Comment (at-susie) [reply src/container/styles.scss:14]: That's so true. I'll update.
Another option that I was thinking but with a hesitation is to reduce the border-radius size from the media wrapper `.media` like `border-top-left-radius: calc(awsui.$border-radius-container - 1px);`.

**Review by just-boris — APPROVED** [review]
The change looks good, but let's wait until after holidays to merge this

**Review by timogasda — APPROVED** [review]

**Pull request merged**
